src/components/AboutLanding/index.tsx

```diff
@@ -0,0 +1,330 @@
+import { Trans } from '@lingui/macro'
+import { Trace, TraceEvent } from '@uniswap/analytics'
+import { BrowserEvent, ElementName, EventName, PageName } from '@uniswap/analytics-events'
+import { BaseButton } from 'components/Button'
+import Card, { CardType } from 'pages/About/Card'
+import { MAIN_CARDS, MORE_CARDS } from 'pages/About/constants'
+import ProtocolBanner from 'pages/About/ProtocolBanner'
+import Swap from 'pages/Swap'
+import { useRef } from 'react'
+import { ArrowDownCircle } from 'react-feather'
+import { useLocation } from 'react-router-dom'
+import { Link as NativeLink } from 'react-router-dom'
+import { useIsDarkMode } from 'state/user/hooks'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
+import { Z_INDEX } from 'theme/zIndex'
+
+const PageContainer = styled.div<{ isDarkMode: boolean }>`
+  position: absolute;
+  top: 0;
+  padding: ${({ theme }) => theme.navHeight}px 0px 0px 0px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+
+  background: ${({ isDarkMode }) =>
+    isDarkMode
+      ? 'linear-gradient(rgba(8, 10, 24, 0) 0%, rgb(8 10 24 / 100%) 45%)'
+      : 'linear-gradient(rgba(255, 255, 255, 0) 0%, rgb(255 255 255 /100%) 45%)'};
+`
+
+const Gradient = styled.div<{ isDarkMode: boolean }>`
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  background: ${({ isDarkMode }) =>
```

Comment:
think this won't quite work anymore, since we want the gradient to end higher up. this was based on the height not increasing past 100vh iirc.